assets/sass/components/_image.scss

```diff
@@ -45,8 +45,20 @@
 
 			img {
 				width: 100%;
+				height: 200px;
+				object-fit: cover;
+				object-position: center;
+			
+				@include breakpoint(medium) {
```

Comment:
there may be a cleaner way to do this, but I wanted the images to be a bit more responsive. on larger screens a lot of the cake image was cut off.